AirCastingTests/BluetoothProtectorTests.swift

```diff
@@ -0,0 +1,199 @@
+import XCTest
+import Resolver
+import Foundation
+@testable import AirCasting
+
+class BluetoothProtectorTests: ACTestCase {
+    
+    func test_whenAskedToPredicateStatuses_excludesUnsupported() throws {
```

Comment:
I don't get the name. 😢

Reply:
agreed. This test needs to be broken doen into two. One for correct statuses and one for incorrect ones. Also better naming would be great. Try to put yourself on a position of someone who don't know the implementation details, just runs the tests to check if they messed something up. If a test fails, it should communicate what is broken just from reading the name (ideally, not always possible. But most of the time). Also - a single test should fail only for a single reason. This test will fail if either the predicate excludes correct status or it includes incorrect ones - this is why we need to split it here